### ISSUE TITLE
Segfault on Raspberry Pi Zero W

### ISSUE THREAD

**Issue by GwennG** [aler9/rtsp-simple-server#1513]
## Which version are you using?

v0.21.5

## Which operating system are you using?

- [ ] Linux amd64 standard
- [ ] Linux amd64 Docker
- [ ] Linux arm64 standard
- [ ] Linux arm64 Docker
- [ ] Linux arm7 standard
- [ ] Linux arm7 Docker
- [ ] Linux arm6 standard
- [X] Linux arm6 Docker
- [ ] Windows amd64 standard
- [ ] Windows amd64 Docker (WSL backend)
- [ ] macOS amd64 standard
- [ ] macOS amd64 Docker
- [ ] Other (please describe)

Raspberry Pi OS Lite
Debian Bullseye 11.6
Docker 23.0.1
libcamera0 0~git20230124+9b860a66-1

```
$ docker image ls aler9/rtsp-simple-server
REPOSITORY                 TAG          IMAGE ID       CREATED      SIZE
aler9/rtsp-simple-server   latest-rpi   9bbd1b3c276a   6 days ago   361MB
```

## Describe the issue

I'm getting the same segfault issue as in #1346 on a Raspberry Pi Zero W running docker image `aler9/rtsp-simple-server:latest-rpi`:
```
INF [path cam] [rpicamera source] ERR: signal: segmentation fault (core dumped)
```

## Describe how to replicate the issue

```
docker run --rm -it --network=host --privileged --tmpfs /dev/shm:exec -v /run/udev:/run/udev:ro -e RTSP_PATHS_CAM_SOURCE=rpiCamera aler9/rtsp-simple-server:latest-rpi
```

## Did you attach the server logs?

```
2023/02/28 13:49:05 INF rtsp-simple-server v0.21.5
2023/02/28 13:49:05 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2023/02/28 13:49:05 INF [RTMP] listener opened on :1935
2023/02/28 13:49:05 INF [path cam] [rpicamera source] started
2023/02/28 13:49:05 INF [HLS] listener opened on :8888
2023/02/28 13:49:05 INF [WebRTC] listener opened on :8889 (HTTP)
2023/02/28 13:49:05 INF [path cam] [rpicamera source] ERR: signal: segmentation fault (core dumped)
2023/02/28 13:49:10 INF [path cam] [rpicamera source] ERR: signal: segmentation fault (core dumped)
2023/02/28 13:49:15 INF [path cam] [rpicamera source] ERR: signal: segmentation fault (core dumped)
```

When setting `logLevel` to `debug` I only get two more lines:
```
DEB [path cam] created
DEB path manager created
```

`libcamera` seems to detect the Raspberry Pi Camera:
```
$ libcamera-hello --list-cameras
Available cameras
-----------------
0 : ov5647 [2592x1944] (/base/soc/i2c0mux/i2c@1/ov5647@36)
    Modes: 'SGBRG10_CSI2P' : 640x480 [58.92 fps - (16, 0)/2560x1920 crop]
                             1296x972 [43.25 fps - (0, 0)/2592x1944 crop]
                             1920x1080 [30.62 fps - (348, 434)/1928x1080 crop]
                             2592x1944 [15.63 fps - (0, 0)/2592x1944 crop]
```

## Did you attach a network dump?

no

**Comment by aler9** [comment]
Hello, after an investigation it turned out that this issue is present when using the Docker image on Raspberry Pis model previous than 3.

The issue is that the base Docker image used in this project is `balenalib/raspberrypi3`, which is not compatible with older Raspberry Pis models, as it uses some ARM instructions that are not available on previous models.

I'll try to switch the base image to `balenalib/raspberry-pi`.

**Comment by aler9** [comment]
The base image has been changed. This will be effective from the next release.

**Comment by aler9** [comment]
fixed in v0.21.6
